src/LIBRETRO/libretro.c

```diff
@@ -293,7 +293,9 @@ static void handle_input(void)
 
    /* Function keys */
    for (i = 0; i < 5; i++)
-      handle_key(KEY88_F6 + i, RETROK_F1 + i);
+      handle_key(KEY88_F1 + i, RETROK_F1 + i);
+   for (i = 0; i < 5; i++)
+      handle_key(KEY88_F6 + i, RETROK_F6 + i);
```

Comment:
KEY88_F5 is not contiguous with KEY88_F6:
https://github.com/libretro/quasi88-libretro/blob/ff94d9d2884d71580c163fcd71b6850c0f393f44/src/keyboard.h#L321-L325
https://github.com/libretro/quasi88-libretro/blob/ff94d9d2884d71580c163fcd71b6850c0f393f44/src/keyboard.h#L342-L346